adafruit_azureiot/__init__.py

```diff
@@ -19,15 +19,16 @@
 * Adafruit CircuitPython firmware for the supported boards:
   https://github.com/adafruit/circuitpython/releases
 
-**ESP32SPI Peripheral Networking**
+**With ESP32SPI Peripheral Networking**
```

Comment:
Change to: "ESP32 AirLift Networking", it has name recognition across our documentation on Learn.